cdc/processor/processor.go

```diff
@@ -493,7 +494,6 @@ func (p *processor) Tick(ctx cdcContext.Context, state *orchestrator.ChangefeedR
 		return state, nil
 	}
 	startTime := time.Now()
-	p.changefeed = state
```

Comment:
could you explain why this line here would cause panic?

Reply:
An error may return in line:494, and then the `p.Close()` would be called by `processorManager`, and we use p.changefeed.ID to log the error message, so the panic occurs.
The safe way is use `p.changefeedID` instead of `pd.changefeed.ID`.